LethalAchievements/Events/Patches/EnemyDamageSource.cs

```diff
@@ -11,17 +10,14 @@ namespace LethalAchievements.Events.Patches;
 /// </summary>
 internal static class EnemyDamageSource
 {
-    /// <summary>
-    ///     The last enemy which damaged a player this frame.
-    /// </summary>
-    public static EnemyAI? CurrentEnemy => Time.frameCount > _currentEnemyHitFrame ? null : _currentEnemy;
-
     private static EnemyAI? _currentEnemy;
     private static int _currentEnemyHitFrame = int.MinValue;
 
-    private static readonly MethodInfo _enemyDamagePathMethod = AccessTools.Method(typeof(EnemyDamageSource), nameof(EnemyDamagePatch));
+    private static readonly MethodInfo EnemyDamagePathMethod =
```

Comment:
MSDN convention

Reply:
According to [this](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions) it should technically be prefixed with s_

Reply:
Also just noticed theres a typo 😄. It's supposed to be patch, not path.

Reply:
> According to [this](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions) it should technically be prefixed with s_

Hmm, it seems like [static readonly are a matter of debate](https://stackoverflow.com/questions/4577358/private-static-readonly-field-capitalization). [StyleCop implies it should be capitalised](https://github.com/StyleCop/StyleCop/blob/master/Project/Docs/Rules/SA1311.html), however. Since the rest of the project adheres to StyleCop (implicitly, through Rider's default configurations), I feel it's best to stick to this convention throughout this project for consistency.

Reply:
> Also just noticed theres a typo 😄. It's supposed to be patch, not path.

Do you want me to fix this in this PR?